core/controllers/editor.py

```diff
@@ -352,28 +352,12 @@ def put(self, exploration_id):
     @require_editor
     def delete(self, exploration_id):
         """Deletes the given exploration."""
-        role = self.request.get('role')
-        if not role:
-            role = None
-
-        if role == rights_manager.ROLE_ADMIN:
-            if not self.is_admin:
-                logging.error(
-                    '%s tried to delete an exploration, but is not an admin.'
-                    % self.user_id)
-                raise self.UnauthorizedUserException(
-                    'User %s does not have permissions to delete exploration '
-                    '%s' % (self.user_id, exploration_id))
-        elif role == rights_manager.ROLE_MODERATOR:
-            if not self.is_moderator:
-                logging.error(
-                    '%s tried to delete an exploration, but is not a '
-                    'moderator.' % self.user_id)
-                raise self.UnauthorizedUserException(
-                    'User %s does not have permissions to delete exploration '
-                    '%s' % (self.user_id, exploration_id))
-        elif role is not None:
-            raise self.InvalidInputException('Invalid role: %s' % role)
+
+        role = ''
```

Comment:
I changed 'role' to 'role_description'. 

But, I am having problem with writing test for such case.
- Suppose, I replace logging.info with another function (say Func) that returns the string given to it. Now, this Func is called from inside of delete() so how can I assert on it's return value ?
- otherwise, if I create the logging.info's argument as a separate variable, that variable is local to the delete method and I can not access it to assert it's value.

I think there might be a better way of doing what I'm trying to do. I googled and couldn't find much help either. 
can you suggest some other way to approach this, thanks ?